src/messaging/Message.spec.ts

```diff
@@ -104,17 +104,31 @@ describe('Messaging', () => {
   it('verify message sender is owner', () => {
     const requestAttestationBody: IRequestAttestationForClaim = {
       content: {
-        delegationId: null,
-        ctypeHash: { nonce: '0x12345678', hash: '0x12345678' },
-        claim: {
-          cType: '0x12345678',
-          owner: identityAlice.getPublicIdentity().address,
-          contents: {},
+        quote: {
+          cost: {
+            tax: 23.45,
+            net: 23.4,
+            gross: 23.5,
+          },
+          currency: 'Euro',
+          acceptance: '0x12345678', // Signature of claimer
+          termsAndConditions: 'www.PDFofExampleTerms&Conditions.com',
+          offerTimeframe: '23.12.2019',
         },
-        claimHashTree: {},
-        legitimations: [],
-        hash: '0x12345678',
-        claimerSignature: '0x12345678',
+        requestForAttestation: {
+          delegationId: null,
+          ctypeHash: { nonce: '0x12345678', hash: '0x12345678' },
+          claim: {
+            cType: '0x12345678',
+            owner: identityAlice.getPublicIdentity().address,
+            contents: {},
+          },
+          claimHashTree: {},
+          legitimations: [],
+          hash: '0x12345678',
+          claimerSignature: '0x12345678',
+        },
+        prerequisiteClaims: ['0x12345678', '0x12345678'],
```

Comment:
Some general testing of the types. 

It passed and I think I should add a new test to check the flow.